scalableminds/webknossos
# Removed "advanced tracing options" flag

As a last remnant of the MTurk integration, the "advanced tracings options" flag was still around. When activated a number of UI elements were hidden and a minimal tracing controller with fewer keyboard controls was loaded.

This PR removes all of that. 

### Steps to test:
- Check that tracing mode is still working
- Create a new TaskType and check that the "Advanced Tracking Options" checkbox is no longer there

### Issues:
- fixes #2297

------
